src/servers/Cargo.toml

```
@@ -58,7 +58,7 @@ openmetrics-parser = "0.4"
 opensrv-mysql = { git = "https://github.com/MichaelScofield/opensrv.git", rev = "1676c1d" }
 opentelemetry-proto.workspace = true
 parking_lot = "0.12"
-pgwire = "0.18"
+pgwire = "0.19.1"
```

non-blocking: "0.19"?
```suggestion
pgwire = "0.19"
```

---

Because 0.19.0 has known bug so we require 0.19.1 as minimum